bigframes/ml/loader.py

```diff
@@ -84,6 +96,16 @@ def _model_from_bq(session: bigframes.Session, bq_model: bigquery.Model):
         return _BQML_MODEL_TYPE_MAPPING[bq_model.model_type]._from_bq(  # type: ignore
             session=session, model=bq_model
         )
+    if (
+        bq_model.model_type == "MODEL_TYPE_UNSPECIFIED"
+        and bq_model._properties["remoteModelInfo"]["endpoint"] is not None
+    ):
+        # Parse the remote model endpoint
+        bqml_endpoint = bq_model._properties["remoteModelInfo"]["endpoint"]
+        endpoint_model = bqml_endpoint.split("/")[-1]
+        return _BQML_ENDPOINT_TYPE_MAPPING[endpoint_model]._from_bq(  # type: ignore
```

Comment:
`endpoint_model` could technically be empty at this point. may be `not bq_model._properties["remoteModelInfo"]["endpoint"]` instead of `bq_model._properties["remoteModelInfo"]["endpoint"] is not None`?

Reply:
Moreover, [] will raise errors when the key doesn't exist, which isn't useful to user. Use dict.get(). 

```
bq_model._properties.get("remoteModelInfo") and bq_model._properties.get("remoteModelInfo").get("endpoint")
```

https://stackoverflow.com/questions/11041405/why-dict-getkey-instead-of-dictkey

Reply:
Done. Thanks for the note!